### PR TITLE
BUG set BUILD env var

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
cc @isuruf

File: recipe/activate-binutils.sh [hunk 1]
```diff
@@ -97,7 +107,8 @@ fi
 
 _tc_activation \
   activate host @CHOST@ @CHOST@- \
-  addr2line ar as c++filt elfedit gprof ld ${GOLD_USED} nm objcopy objdump ranlib readelf size strings strip
+  addr2line ar as c++filt elfedit gprof ld ${GOLD_USED} nm objcopy objdump ranlib readelf size strings strip \
+  BUILD,${BUILD}
```
Comment (isuruf) [inline recipe/activate-binutils.sh:111]: Since these are native compilers, we can just do `BUILD,@CHOST@`

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by beckermr** [comment]
ok @isuruf - this should be ready to merge!

**Pull request merged**